getting-started/revo-to-full-manual.txt

```diff
@@ -1,122 +1,217 @@
+    "color": 39423,
+    "fields": [
+      {
+        "name": "__Getting Started__",
+        "value": "⬥ <#1020546518941454438>\n⬥ <#1020546905865986068>\n⬥ <#1020547206303985714>",
+        "inline": true
+      },
+      {
+        "name": "__DPM-Advice Channels__",
+        "value": "⬥ <#1020729755064074291>\n⬥ <#1020729582086782996>\n⬥ <#1020729671127662643>",
+        "inline": true
+      },
+      {
+        "name": "__Help Channels__",
+        "value": "⬥ <#656898197561802760>\n⬥ <#1019648841965916220>",
+        "inline": true
+      }
+    ]
+  }
+}
+.embed:json
 
 .
-⬥ **__Ultimate abilities__**
-    • Should normally be used manually even if using Revolution (except afk slayer).
-    • Good Revolution bars will have the stronger abilities towards the beginning of the revo bar so they get activated more frequently.
-        - This is good for general usage but detrimental for damage boosting ultimates like Sunshine <:sunshine:994921119381463070>, Death's Swiftness <:deathsswift:994921434633744455> and Berserk <:zerk:535532854004678657>.
-    • The best abilities will likely be on cooldown at the start of your damage boosting ult due to being at the front of the revo bar.
-    • Some of the best abilities are bleeds which will be near the front of the revo bar, but bleeds are NOT boosted by damage boosting ultimates.
-    • Full manual benefits a lot in damage boosting ultimates because you have full control of when you use the strong abilities.
-        - Strong abilities can be saved until you are within the damage boosting ultimate to get more out of them.
-        - Bleed usage can be easily avoided in damage boosting ultimates since you can just choose to not input any bleed abilities.
+> __**Revolution (Revo)**__
+.tag:revo
+*Note: For a list of of revo/revo++bars, see `!revo`, `!revo++` and `!aoe`.*
+__**What Is Revolution**__
+⬥ **Revolution lets the game automatically activate abilities on your ability bar in sequential order** (based on adrenaline and cooldowns).
+⬥ Revo can be configured to activate any combination of:
+    • Basics
+    • Thresholds
+    • Ultimates
+⬥ When it activates also thresholds/ultimates for you, it is referred to as **revo++**.
+⬥ **Remember, you can still provide user input to override revo/revo++ when you want**.
+    • This may be easier with ability queueing on.
 
-⬥ The gif below will demonstrate how Revolution can make someone use bad abilities in a damage boosting ultimate. Take a look at what abilities were used right after sunshine:
 .
-https://i.imgur.com/Y6XWNaF.jpg
+__**Is Revolution Good?**__
+⬥ It gives up some control over using your basics but also means less user input is required.
+⬥ It's good if:
+    • You're just getting into PvM
+    • Combat seems to have too much to focus on
+    • You want the game to use basics but still have control over stronger thresholds/etc.
+    • etc.
```

Comment:
Would get rid of this and just preface with "Some reasons revo can be helpful include:"